osf/migrations/0195_update_schemas.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-11-13 16:11
+from __future__ import unicode_literals
+
+from django.db import migrations
+from osf.utils.migrations import UpdateRegistrationSchemas
+
+def make_egap_active_but_invisible(state, schema):
+        RegistrationSchema = state.get_model('osf', 'registrationschema')
+        egap_registration = RegistrationSchema.objects.get(name='EGAP Registration')
```

Comment:
is there not a better way to identify the registration? What if the name changes?

Reply:
alas, `(name, schema_version)` is the unique constraint.
as long as that is true, we should not ever change their names.
yes, this is bad.